tomlkit/container.py

```diff
@@ -46,6 +46,25 @@ def __init__(self, parsed: bool = False) -> None:
     def body(self) -> List[Tuple[Optional[Key], Item]]:
         return self._body
 
+    def unwrap(self, recursive: bool = True) -> str:
+        unwrapped = {}
+        for k, v in self._body:
```

Comment:
We don't need to iterate over the `_body`, `for k, v in self.items()` can retrieve all key and value pairs, without any format control elements.